sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -25,6 +25,13 @@ have to be online at the same time.
   - [Authenticate the client](#authenticate-the-client)
 - [Key concepts](#key-concepts)
 - [Examples](#examples)
+  - [Create a sender or receiver using connection string](#create-a-sender-or-receiver-using-connection-string)
+  - [Send Message to Queue or Topic](#send-message-to-queue-or-topic)
+  - [Receive message from Queue or Subscription](#receive-message-from-queue-or-subscription)
+  - [Peek message from Queue or Subscription](#peek-message-from-queue-or-subscription)
```

Comment:
Send and receive using AAD and connection string are good scenarios. We don't need to duplicate it with peek (this is also a diagnostic scenario we don't want to put to the forefront) and other small operations. 

It makes the table of contents long. (Feedback we received in EH)